hooks/pre-commit

```diff
@@ -4,7 +4,7 @@ LC_ALL=C
 GIT_ROOT="$(git rev-parse --show-toplevel)"
 
 LOCAL_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
-VALID_BRANCH_REGEX="^(qa|style|chore|refactor|feat|internal|bugfix|improvement|library|prerelease|release|hotfix)\/[a-zA-Z0-9._-]+$"
+VALID_BRANCH_REGEX="^(qa|style|chore|refactor|feat|internal|bugfix|improvement|library|docs|prerelease|release|hotfix)\/[a-zA-Z0-9._-]+$"
```

Comment:
we should also accept `docs: ...`

Reply:
I don't really think this check is necessary.

Reply:
lets just remove it